wagtail/images/tests/test_admin_views.py

```diff
@@ -78,6 +78,18 @@ def test_search(self):
         self.assertContains(response, "a cute kitten")
         self.assertNotContains(response, "a cute puppy")
 
+        global_search_response = self.get({"q": "cut"})
+        chooser_search_response = self.client.get(
+            reverse("wagtailimages_chooser:choose") + "?q=cut"
+        )
```

Comment:
I think it's better to directly check the exact results returned by a partial match, rather than checking "is it the same result set as the chooser view", since the latter would pass if both views returned the same set of bogus results. Everything else here looks good, so I'll update this when I merge.

Reply:
Actually, it's worth noting that the tests only currently run against the fallback search backend, which always performs partial word matches - so this test would pass even without the fix in place. But dealing with that is a separate matter that's out of scope here (#9901 / #9903 / #10208)...

Reply:
yea i noticed that, that's why i said `not sure if there's a better way to test it though`